internal/cli/up.go

```diff
@@ -83,9 +87,28 @@ In that case, see https://tilt.dev/user_config.html and/or comments in your Tilt
 	cmd.Flags().BoolVar(&noBrowser, "no-browser", false, "If true, web UI will not open on startup.")
 	cmd.Flags().StringVar(&c.outputSnapshotOnExit, "output-snapshot-on-exit", "", "If specified, Tilt will dump a snapshot of its state to the specified path when it exits")
 
+	// this is to test the new behavior before enabling it in Tilt 1.0
+	// https://app.clubhouse.io/windmill/epic/5549/make-tui-hard-to-find-in-tilt-1-0
+	cmd.Flags().BoolVar(&c.defaultTUI, "default-tui", true, "If false, we'll hide the TUI by default")
```

Comment:
i'm surprised this isn't called --default-hud or similar, given that we don't really use the word "tui" elsewhere in the code, tho I guess if this is going away it doesn't matter

Reply:
changed!